packages/devtools_app_shared/lib/src/ui/theme/ide_theme.dart

```diff
@@ -24,18 +24,20 @@ final class IdeTheme {
     bool? isDarkMode,
   }) : _isDarkMode = isDarkMode;
 
-  final Color? backgroundColor;
-  final Color? foregroundColor;
-  final double fontSize;
+  Color? backgroundColor;
+  Color? foregroundColor;
+  double fontSize;
   final EmbedMode embedMode;
-  final bool? _isDarkMode;
+  bool? _isDarkMode;
 
   double get fontSizeFactor => fontSize / unscaledDefaultFontSize;
 
   bool get embedded => embedMode.embedded;
 
   bool get isDarkMode => _isDarkMode ?? useDarkThemeAsDefault;
 
+  set isDarkMode(bool newIsDarkMode) => _isDarkMode = newIsDarkMode;
```

Comment:
I think we can get rid of this setter. I don't see that it is being used anywhere